### DISCUSSION TITLE
Question: Is there a way to support shadows in the theme?

### DISCUSSION THREAD

**Discussion by zlobizlobi** [tamagui/tamagui#1169] [General]
I was wondering whether it's currently possible to support shadows in the theme? I was not able to find anything about this in the documentation.

**Comment by mattsoltani** [comment]
We are also interested in this as a part of our design system implementation. 
I could also see the need for border, animation and more. What about allowing for custom tokens, so one can pull these out when using the `getTokens` method?

**Reply by maylukas** [reply]
We are also looking into storing a harmonized version of shadows in the theme. Currently we are storing them in constants and are spreading them onto the respective Stack components which is okay but not ideal. Preferably we'd have a "shadow" property that would define a set of styles to be set for the shadow.
